[embassy-boot]: compile failed with "log" feature

I'm creating bootloader for a custom cortex-m4 board using embassy-boot, and it compiled failed when enable ```log``` features.
cargo.toml:
```toml
[package]
name = "bootloader"
version = "0.1.0"
edition = "2024"

[dependencies]
bsp = { path = "../bsp" }
common = { version = "0.1.0", path = "../common" }
cortex-m = { version = "0.7.7", features = ["critical-section-single-core"] }
cortex-m-rt = { version = "0.7.5", features = ["device"] }
embassy-boot = { version = "0.6.1", features = ["ed25519-salty", "log"] }
embassy-sync = "0.7.2"
embassy-time = "0.5.0"
embedded-io-async = "0.6.1"
embedded-storage = "0.3.1"
log = "0.4.27"
```
error message:
``` text
Compiling embassy-boot v0.6.1
error[E0277]: `[u8; 32]` doesn't implement `Display`
   --> /home/cc/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/embassy-boot-0.6.1/src/firmware_updater/asynch.rs:133:17
    |
132 |                 "Verifying with public key {}, signature {} and message {} yields ok: {}",
    |                                            -- required by this formatting parameter
133 |                 public_key.to_bytes(),
    |                 ^^^^^^^^^^^^^^^^^^^^^ `[u8; 32]` cannot be formatted with the default formatter
    |
    = help: the trait `Display` is not implemented for `[u8; 32]`
    = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
    = note: this error originates in the macro `$crate::__private_api::format_args` which comes from the expansion of the macro `trace` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: `[u8; 64]` doesn't implement `Display`
   --> /home/cc/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/embassy-boot-0.6.1/src/firmware_updater/asynch.rs:134:17
    |
132 |                 "Verifying with public key {}, signature {} and message {} yields ok: {}",
    |                                                          -- required by this formatting parameter
133 |                 public_key.to_bytes(),
134 |                 signature.to_bytes(),
    |                 ^^^^^^^^^^^^^^^^^^^^ `[u8; 64]` cannot be formatted with the default formatter
    |
    = help: the trait `Display` is not implemented for `[u8; 64]`
    = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
    = note: this error originates in the macro `$crate::__private_api::format_args` which comes from the expansion of the macro `trace` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: `[u8; 32]` doesn't implement `Display`
   --> /home/cc/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/embassy-boot-0.6.1/src/firmware_updater/blocking.rs:168:17
    |
167 |                 "Verifying with public key {}, signature {} and message {} yields ok: {}",
    |                                            -- required by this formatting parameter
168 |                 public_key.to_bytes(),
    |                 ^^^^^^^^^^^^^^^^^^^^^ `[u8; 32]` cannot be formatted with the default formatter
    |
    = help: the trait `Display` is not implemented for `[u8; 32]`
    = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
    = note: this error originates in the macro `$crate::__private_api::format_args` which comes from the expansion of the macro `trace` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: `[u8; 64]` doesn't implement `Display`
   --> /home/cc/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/embassy-boot-0.6.1/src/firmware_updater/blocking.rs:169:17
    |
167 |                 "Verifying with public key {}, signature {} and message {} yields ok: {}",
    |                                                          -- required by this formatting parameter
168 |                 public_key.to_bytes(),
169 |                 signature.to_bytes(),
    |                 ^^^^^^^^^^^^^^^^^^^^ `[u8; 64]` cannot be formatted with the default formatter
    |
    = help: the trait `Display` is not implemented for `[u8; 64]`
    = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
    = note: this error originates in the macro `$crate::__private_api::format_args` which comes from the expansion of the macro `trace` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0277`.
error: could not compile `embassy-boot` (lib) due to 4 previous errors
```

--- a/embassy-boot/src/firmware_updater/asynch.rs
+++ b/embassy-boot/src/firmware_updater/asynch.rs
@@ -115,6 +115,7 @@ impl<'d, DFU: NorFlash, STATE: NorFlash> FirmwareUpdater<'d, DFU, STATE> {
             use salty::{PublicKey, Signature};
 
             use crate::digest_adapters::salty::Sha512;
+            use crate::fmt::Bytes;
 
             fn into_signature_error<E>(_: E) -> FirmwareUpdaterError {
                 FirmwareUpdaterError::Signature(signature::Error::default())
@@ -130,9 +131,9 @@ impl<'d, DFU: NorFlash, STATE: NorFlash> FirmwareUpdater<'d, DFU, STATE> {
             let r = public_key.verify(&message, &signature);
             trace!(
                 "Verifying with public key {}, signature {} and message {} yields ok: {}",
-                public_key.to_bytes(),
-                signature.to_bytes(),
-                message,
+                Bytes(&public_key.to_bytes()),
+                Bytes(&signature.to_bytes()),
+                Bytes(&message),
                 r.is_ok()
             );
             r.map_err(into_signature_error)?;

--- a/embassy-boot/src/firmware_updater/blocking.rs
+++ b/embassy-boot/src/firmware_updater/blocking.rs
@@ -150,6 +150,7 @@ impl<'d, DFU: NorFlash, STATE: NorFlash> BlockingFirmwareUpdater<'d, DFU, STATE>
             use salty::{PublicKey, Signature};
 
             use crate::digest_adapters::salty::Sha512;
+            use crate::fmt::Bytes;
 
             fn into_signature_error<E>(_: E) -> FirmwareUpdaterError {
                 FirmwareUpdaterError::Signature(signature::Error::default())
@@ -165,9 +166,9 @@ impl<'d, DFU: NorFlash, STATE: NorFlash> BlockingFirmwareUpdater<'d, DFU, STATE>
             let r = public_key.verify(&message, &signature);
             trace!(
                 "Verifying with public key {}, signature {} and message {} yields ok: {}",
-                public_key.to_bytes(),
-                signature.to_bytes(),
-                message,
+                Bytes(&public_key.to_bytes()),
+                Bytes(&signature.to_bytes()),
+                Bytes(&message),
                 r.is_ok()
             );
             r.map_err(into_signature_error)?;

